cmd/puppeth/module_bootnode.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2017 The go-ethereum Authors
+// This file is part of go-ethereum.
+//
+// go-ethereum is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// go-ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with go-ethereum. If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/Onther-Tech/plasma-evm/log"
+	"html/template"
+	"math/rand"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+// faucetDockerfile is the Dockerfile required to build a faucet container to
+// grant crypto tokens based on GitHub authentications.
+var bootnodeDockerfile = `
+FROM {{.Image}}
+
+EXPOSE 30301 30301/udp
+
+WORKDIR /usr/local/bin
```

Comment:
Like other puppeth component, use default working directory ($HOME), to make it easy to find execution script file when attached to docker container

Reply:
`sealer` container running script , `geth.sh`, placed at `/usr/local/bin`.
I though `bootnode` also kinda node like, sealer and usernode. so I placed it.

btw, There is no puppeth components in placed its running script at ($HOME, /root).
or Just insert container's entry-point, like faucet container. 

https://github.com/Onther-Tech/plasma-evm/blob/a83904733a0ef3c83b72cc795955b035352db494/cmd/puppeth/module_faucet.go#L44-L49

I think your suggestion is best for common *nix user. but not for who familiar with puppeth.
Do you think that it makes easier to find that script if placed `/root'? 
I don't really know which place would be better, let me notice :)

Reply:
Sorry, default `WORKDIR` is `/`, not `$HOME`.

My comment meant that docker files of other puppeth components don't have `WORKDIR` instruction, they make executable file at root directory (`/server.js` for [module_dashboard.go](https://github.com/Onther-Tech/plasma-evm/blob/e205e899999f21e9a565c8279a626342449bf9cb/cmd/puppeth/module_dashboard.go#L540), `geth.sh` for [module_node.go](https://github.com/Onther-Tech/plasma-evm/blob/e205e899999f21e9a565c8279a626342449bf9cb/cmd/puppeth/module_node.go#L56))

So the executable of boot node also should be placed at `/`, like others.

Reply:
I see, It make sense!
I totally agress with use `/` as `WORKDIR`.
Will updated it.

Reply:
Updated & Tested with https://github.com/Onther-Tech/plasma-evm/pull/98/commits/1bb3a1602fb165f8828550f825306e7513776b48